packages/vitest-environment-nuxt/src/index.ts

```diff
@@ -61,6 +59,7 @@ export default <Environment>{
     }
 
     win.$fetch = createFetch({ fetch: win.fetch, Headers: win.Headers as any })
+    win.scrollTo = () => {}
```

Comment:
Why is this needed? 😄

Reply:
JSDom doesn't have `scrollTo` implemented.